src/Elmish/React/DOM.purs

```diff
@@ -20,3 +27,6 @@ fragment :: Array ReactElement -> ReactElement
 fragment = createElement fragment_ {}
 
 foreign import fragment_ :: ReactComponent {}
+
+foreign import render :: ReactElement -> Element -> Effect Unit
```

Comment:
These functions should not be in the same module as `empty` or `fragment`.
These functions are very low-level, almost never used by normal code. So they would needlessly pollute the namespace.

Reply:
Sure, but they belong to `react-dom`. Since Elmish assumes basic knowledge of React, I don’t see why we would have different naming, signatures, and modules for some basic functionality. If we don’t want to make it first-class, we could move it into `Elmish.React.DOM.Internal`.